bytes/bytes.gobra

```diff
@@ -0,0 +1,33 @@
+//+gobra
+
+package bytes
+
+	
+ghost
+requires count >= 0
+decreases count
+pure func Repeat(b seq[byte], count int) (res seq[byte]) {
+	return count == 0 ? seq[byte]{} : ( b ++ Repeat(b, count - 1) )
+}
+
+ghost
+requires 0 < len(sep)
+decreases
+pure func Split(b, sep seq[byte]) (res seq[seq[byte]]) {
+	return SplitInner(b, sep, seq[byte]{})
+}
+
+ghost
+requires 0 < len(sep)
+decreases len(s)
+pure func SplitInner(s, sep, ac seq[byte]) (res seq[seq[byte]]) {
+	return len(s) == 0 ?
+		( len(ac) == 0 ?
+			seq[seq[byte]]{} :
+			seq[seq[byte]]{ac}) :
+		( sep == s ?
+			seq[seq[byte]]{ ac, seq[byte]{} } :
```

Comment:
why do we put `seq[byte]{}` in the result here, but not when the accumulator is empty in the base case?

Reply:
Good point, that is a bug. `bytes.Split` returns `[[]]` when splitting an empty slice.